test/checks/mobile/target-size.js

```diff
@@ -39,45 +38,139 @@ describe('target-size tests', function () {
+          '">x</a>' +
+          '<div style="' +
+          'display: inline-block; width:20px; height:20px; margin-left: -20px;' +
+          '">x</div>'
+      );
+      assert.isTrue(check.evaluate.apply(checkContext, checkArgs));
+      assert.deepEqual(checkContext._data, { messageKey: 'obscured' });
+    });
+
+    it('returns true when obscured by another focusable widget', function () {
+      var checkArgs = checkSetup(
+        '<a href="#" id="target" style="' +
+          'display: inline-block; width:20px; height:20px;' +
+          '">x</a>' +
+          '<a href="#" style="' +
+          'display: inline-block; width:20px; height:20px; margin-left: -20px;' +
+          '">x</a>'
+      );
+      assert.isTrue(check.evaluate.apply(checkContext, checkArgs));
+      assert.deepEqual(checkContext._data, {
+        messageKey: 'obscured',
+        minSize: 24,
+        width: 20,
+        height: 20
+      });
+    });
+
+    it('ignores obscuring element has pointer-events:none', function () {
+      var checkArgs = checkSetup(
+        '<a href="#" id="target" style="' +
+          'display: inline-block; width:20px; height:20px;' +
+          '">x</a>' +
+          '<span style="' +
+          'display: inline-block; pointer-events: none; width:20px; height:20px; margin-left: -20px;' +
+          '">x</span>'
+      );
+      assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
+      assert.deepEqual(checkContext._data, {
+        minSize: 24,
+        width: 20,
+        height: 20
+      });
     });
   });
 
-  it('returns false for obscured targets with insufficient space', function () {
-    var checkArgs = checkSetup(
-      '<button id="target" style="' +
-        'display: inline-block; width:40px; height:30px; margin-left:30px;' +
-        '">x</button>' +
-        '<button style="' +
-        'display: inline-block; width:40px; height:30px; margin-left: -10px;' +
-        '">x</button>' +
-        '<button style="' +
-        'display: inline-block; width:40px; height:30px; margin-left: -100px;' +
-        '">x</button>'
-    );
-    assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
-    assert.deepEqual(checkContext._data, {
-      messageKey: 'obscured',
-      minSize: 24,
-      width: 20,
-      height: 30
+  describe('when partially obscured', function () {
+    it('returns true for focusable non-widgets', function () {
+      var checkArgs = checkSetup(
+        '<button id="target" style="' +
+          'display: inline-block; width:40px; height:30px; margin-left:30px;' +
+          '">x</button>' +
+          '<button style="' +
+          'display: inline-block; width:40px; height:30px; margin-left: -10px;' +
+          '">x</button>' +
+          '<span tabindex="0" style="' +
+          'display: inline-block; width:40px; height:30px; margin-left: -100px;' +
+          '">x</span>'
+      );
+      assert.isTrue(check.evaluate.apply(checkContext, checkArgs));
```

Comment:
I feel like this should be `false`. the `span` with `tabindex=0` captures the click on the left side of the target, making the clickable width effectively 20px.

<img width="110" alt="image" src="https://user-images.githubusercontent.com/2433219/191381423-5862f5e6-d84d-4e48-b829-8fec85db89e1.png">

Reply:
This is no different than if that was just a narrower button. If someone accidentally hits the span instead of the button, nothing happens. Or at least, we can't assume something is going to happen. If that span is a widget that is missing a role, a different accessibility violation exists in that page, and after they've fixed that (by adding a role to the span) this rule will work correctly.

We could maybe report this as incomplete? I'd prefer we do this later though. This PR is much larger than I want it to be already. Please open an issue for this.